uktrade/data-hub-frontend
# Add 'no recent export interactions' to notification settings list

## Description of change

Divides the 'Reminders and email notifications settings' page into two sections, one for Investment and one for Exports. The Exports section is visible behind a feature flag. It contains a toggle component titled 'Companies with no recent interaction settings', which: 
- allows users to review their existing notification settings
- contains a link to the edit notifications form (not yet created)

## Test instructions
If running locally, visit: http://localhost:3000/reminders/settings?estimated_land_date=true . You should see the same view as before, except for a new heading 'Investment'.

To view the Exports changes, you will need to add a `export-email-reminders` feature flag, or watch the `test/functional/cypress/specs/reminders/settings/settings-spec.js` Cypress test.

## Screenshots

### Before
<img width="1007" alt="Screenshot 2022-11-02 at 14 25 48" src="https://user-images.githubusercontent.com/23265724/199489115-0ae9f00f-c345-47c3-8e4c-9e3e1350b42a.png">


### After
<img width="673" alt="Screenshot 2022-11-02 at 14 15 11" src="https://user-images.githubusercontent.com/23265724/199489032-ce3680f1-7726-48fe-b5f4-bb933a46426e.png">

## Checklist

[//]: # 'When submitting a PR make sure the code review guidelines have been satisfied.
https://github.com/uktrade/data-hub-frontend/blob/main/docs/Code%20review%20guidelines.md'
